binaries/build_wheels.py

```diff
@@ -26,33 +26,41 @@
 
 # Version constants.
 # Change to download different versions.
-FFMPEG_VERSION = 'n4.4-2'
-PACKAGER_VERSION = 'v2.6.1'
+FFMPEG_VERSION = 'n7.1-1'
+PACKAGER_VERSION = 'v3.2.0'
 
 # A map of suffixes that will be combined with the binary download links
 # to achieve a full download link.  Different suffix for each platform.
 # Extend this dictionary to add more platforms.
 PLATFORM_SUFFIXES = {
-    # 64-bit Windows
-    'win_amd64': '-win-x64.exe',
-    # 64-bit Linux
-    'manylinux1_x86_64': '-linux-x64',
-    # Linux on ARM
+    # Linux x64
+    'manylinux2014_x86_64': '-linux-x64',
+    # Linux arm64
     'manylinux2014_aarch64': '-linux-arm64',
-    # 64-bit with 10.9 SDK
+    # macOS x64 with 10.9 SDK
     'macosx_10_9_x86_64': '-osx-x64',
+    # macOS arm64 with 10.9 SDK
+    'macosx_10_9_arm64': '-osx-arm64',
+    # Windows x64
+    'win_amd64': '-win-x64.exe',
 }
 
 FFMPEG_DL_PREFIX = 'https://github.com/shaka-project/static-ffmpeg-binaries/releases/download/' + FFMPEG_VERSION
 PACKAGER_DL_PREFIX = 'https://github.com/shaka-project/shaka-packager/releases/download/' + PACKAGER_VERSION
 
-# The download links to each binary.  These download links
-# aren't complete, they miss the platfrom-specific suffix.
-BINARIES_DL = [
+# The download links to each binary.  These download links aren't complete.
+# They are missing the platfrom-specific suffix and optional distro-specific
+# suffix (Linux only).
+FFMPEG_BINARIES_DL = [
     FFMPEG_DL_PREFIX + '/ffmpeg',
     FFMPEG_DL_PREFIX + '/ffprobe',
+]
+PACKAGER_BINARIES_DL = [
     PACKAGER_DL_PREFIX + '/packager',
 ]
+UBUNTU_SUFFIXES = map(
+    lambda version: '-ubuntu-{}'.format(version),
```

Comment:
nit: the literal `'-ubuntu-'` is used two times in here. maybe we can fit it inside `_ubuntu_versions_with_hw_encoders`.

Reply:
That would complicate `if distro.version() in _ubuntu_versions_with_hw_encoders:`